src/http_server.rs

```diff
@@ -87,12 +87,10 @@ impl<T: Controller> HttpServer<T> {
             // Taxonomy router paths. Keep in sync with taxonomy_router.rs
             (vec![Method::Get, Method::Post], "api/v1/services".to_owned()),
             (vec![Method::Post, Method::Delete], "api/v1/services/tags".to_owned()),
-            (vec![Method::Get, Method::Post], "api/v1/channels/getters".to_owned()),
-            (vec![Method::Get, Method::Post], "api/v1/channels/setters".to_owned()),
+            (vec![Method::Post], "api/v1/channels".to_owned()),
```

Comment:
Ah you mean ```GET```, right? I've removed ```GET``` and ```POST``` for old endpoints (```channels/getters``` and ```channels/setters```) and added ```POST``` for new endpoint (```channels```), but yeah, we probably still need ```GET``` as well...